### PR TITLE
Freshen up README, remove old ReST file, switch setup.py to set markdown long description

### PR DESCRIPTION
**Pull request by seibert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+* Describe data sets in catalog files for easy reuse and sharing between projects and with others.
+* Share catalog information (and data sets) over the network with the Intake server
+
+Documentation is available at [Read the Docs](http://intake.readthedocs.io/en/latest).
 
 Development Environment
 ----------------------------
  * Create development Python environment.
- * Install [snappy](https://github.com/andrix/python-snappy)
  * `pip install -r requirements.txt`
```
Comment (martindurant) [inline README.md]: Should we also provide an environment.yaml, or a link to the conda recipe?

**Comment by martindurant** [comment]
+1 from me

File: README.md (hunk 1 above)
Comment (seibert) [reply README.md]: yeah, I would not mind setting up an environment.yaml, but we can do that as part of making some more complete developer docs.

**Comment by seibert** [comment]
OK, merging

**Pull request merged**